src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TelemetryChannelEventSource.cs

```diff
@@ -451,6 +451,17 @@ public void TransmissionStorageUnexpectedRetryOfBadFile(string fileName, string
             this.WriteEvent(63, fileName, this.ApplicationName);
         }
 
+        [Event(64, Message = "Transmission locally throttled. Throttle Limit: {0}. Attempted: {1}. Accepted: {2}. ", Level = EventLevel.Warning)]
```

Comment:
I'm not opposed to reducing the level here. It is user actionable if not caused from retrieval from storage (data rate needs to decrease)